app/controllers/flowcommerce_spree/orders_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module FlowcommerceSpree
+  class OrdersController < ApplicationController
+    wrap_parameters false
+
+    skip_before_action :setup_tracking, only: :order_completed
+
```

Comment:
Hi @texpert Why are we adding this endpoint on this Pull Request? It looks like the endpoint for validating the puchase and not dedicated to the spree admin

Reply:
Hi, @davidmejuri !

This PR will be renamed - it is the Phase_3 release, in fact.